pages/agent/v2/osx.md.erb

```diff
@@ -27,7 +27,7 @@ If you don't use Homebrew you should follow the [Linux](/docs/agent/v2/linux) in
 
 ## SSH Key Configuration
 
-SSH keys should be copied to (or generated into) the `.ssh` directory in the users’s home directory (i.e. `/Users/some-user/.ssh`). For example, to generate a new private key which you can add to your source code host:
+SSH keys should be copied to (or generated into) the `.ssh` directory in the users’s home directory (e.g. `/Users/alice/.ssh`). For example, to generate a new private key which you can add to your source code host:
```

Comment:
This one felt halfway between an _example_ and a _clarifying restatement_, so I shifted it more firmly into the _example_ camp.